src/resources/views/base/inc/head.blade.php

```diff
@@ -27,10 +31,3 @@
 
     @yield('after_styles')
     @stack('after_styles')
-
-    {{-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries --}}
-    {{-- WARNING: Respond.js doesn't work if you view the page via file:// --}}
-    <!--[if lt IE 9]>
-    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
-    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
```

Comment:
Why remove the Shim and Respond.js? The way I understand it, this would remove some IE8 support from our "Legacy" theme, which... we don't want. We want people who still depend on IE8 to have a solution in Backpack, even if we don't push any new stuff to it, right?

Reply:
Didn't find a reason to remove this, so I've added it back.